packages/toolpad-app/src/toolpad/Home/index.tsx

```diff
@@ -211,28 +219,47 @@ function CreateAppDialog({ onClose, ...props }: CreateAppDialogProps) {
               onChange={handleDomChange}
             />
           ) : null}
-          {config.recaptchaSiteKey ? (
-            <Typography variant="caption" color="text.secondary">
-              This site is protected by reCAPTCHA and the Google{' '}
-              <Link
-                href="https://policies.google.com/privacy"
-                underline="none"
-                target="_blank"
-                rel="noopener noreferrer"
+          {config.isDemo && latestStoredApp ? (
+            <Box mt={1} sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+              <Typography variant="subtitle2" textAlign="center">
+                OR
+              </Typography>
+              <Button
```

Comment:
This might be subjective, but my initial impression when I saw the popup was that this action was quite "mute", like very easy to miss. If our intention is to lead people to previously created app, maybe we should use a little bit more visual style? I.e. `outlined` variant might be a bit easier to recognise as an action 🤔

Reply:
I didn't want to make it too strong not to look like the submit button of a form, and trying to make both creating a new app or going to the previous one have equal weight...
If we want users to do one of these 2 things more than the other though we could try different things visually.

Reply:
Maybe an outlined button that's a bit narrower than the form will work, I can try that.